spa/src/index.tsx

```diff
@@ -1,39 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import "@atlaskit/css-reset";
-import { setGlobalTheme } from "@atlaskit/tokens";
```

Comment:
This is due to switch to react-script, it emits many more warnings.
This is to extract the separate file so that the hot reload works.